### ISSUE TITLE
Latest Awakened-PoE download flagged as malware

### ISSUE THREAD

**Issue by RetWolf** [SnosMe/awakened-poe-trade#71]
After attempting to download the latest release, I was blocked when downloading both the Windows installer and the portable release. I changed some Windows security settings to download it, but re-enabled them before running the application locally. Application startup was blocked by Windows Defender as being malware, reporting that the Awakened-PoE process was attempting execute a Trojan.

Here is a screenshot of the threat being blocked when downloading the release:
![image](https://user-images.githubusercontent.com/28321156/78570822-7a401600-77f3-11ea-8e64-c88f10a599fe.png)

**Comment by RetWolf** [comment]
Didn't see other related issue, will close this one out and follow there.

**Comment by SnosMe** [comment]
Duplicate of #65
